src/lib/insights/listener.tsx

```diff
@@ -9,23 +9,37 @@ type WithInsightsListenerProps = {
   insights: InsightsClientWrapper;
 };
 
+const findRealTarget = (
```

Comment:
We could name this `findInsightsRoot`.

Reply:
renamed to `findInsightsTarget` in 115c932f as agreed at https://github.com/algolia/instantsearch.js/pull/4197#discussion_r351394715